### DISCUSSION TITLE
With Vercel's free version, for Next.js, how do I know when I have exceeded the allowed traffic for this level?

### DISCUSSION THREAD

**Discussion by SurfWeb3** [vercel/next.js#65190] [Help] [answered]
### Summary

My app suddenly stopped working. The app is an NFT marketplace, for curated art NFTs. However, the NFT images, which were once visible in the app, are no longer shown. The error message is "failed to load metadata". I'm checking to find out if the issue is due to too much traffic for the free version. With Vercel's free version, for Next.js, how do I know when I have exceeded the allowed traffic for this level?

**Comment by wwwdepot** [comment, marked as answer]
See the **Usage** tab in your dashboard

**Reply by SurfWeb3** [reply]
Thanks for the answer!

**Reply by SurfWeb3** [reply]
Thanks for the answer!
